js/bitso.js

```diff
@@ -880,7 +880,7 @@ module.exports = class bitso extends Exchange {
             'fund_currency': currency['id'],
         };
         const response = await this.privateGetFundingDestination (this.extend (request, params));
-        let address = this.safeString (response['payload'], 'account_identifier');
+        let address = this.safeString (response['payload'], 'account_identifier', '');
```

Comment:
The address cannot be an empty string. We're missing a proper parseDepositAddress here, and a checkAddress call, but I would leave that our for a standalone PR. Let's remove this default value.